providers/gce/gce_loadbalancer.go

```diff
@@ -212,6 +219,13 @@ func (g *Cloud) UpdateLoadBalancer(ctx context.Context, clusterName string, svc
 		return err
 	}
 
+	// GCE load balancers do not support services with LoadBalancerClass set. LoadBalancerClass can't be updated for an existing load balancer, so here we don't need to clean any resources.
```

Comment:
same comment as in https://github.com/kubernetes/cloud-provider-gcp/pull/586/files#r1289339510